components/src/dynamo/trtllm/tests/test_trtllm_fpm_publisher.py

```
@@ -0,0 +1,237 @@
+    import asyncio as _asyncio
+
+    real_create_task = _asyncio.create_task
+    _asyncio.create_task = lambda coro: MagicMock(add_done_callback=lambda _: None)
+    try:
+        try:
+            pub.initialize()
+        except Exception:
+            # initialize() touches other subsystems we don't care about; the
+            # FPM-related assertions below tell us whether the construction
+            # (or non-construction) happened as expected.
+            pass
```

Ruff (0.15.10)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)